apps/terragrunt.hcl

```diff
@@ -82,13 +86,18 @@ inputs = {
   core_kv_id              = dependency.core.outputs.core_kv_id
   core_log_analytics_name = dependency.core.outputs.core_log_analytics_name
 
-  transform_feature_store_server_name = dependency.transform.outputs.feature_store_server_name
-  transform_feature_store_db_name     = dependency.transform.outputs.feature_store_db_name
+  transform_feature_store_server_name  = dependency.transform.outputs.feature_store_server_name
+  transform_feature_store_db_name      = dependency.transform.outputs.feature_store_db_name
+  transform_apps_ad_group_display_name = dependency.transform.outputs.apps_ad_group_display_name
+  transform_developers_ad_group_display_name = dependency.transform.outputs.developers_ad_group_display_name
+  transform_apps_ad_group_principal_id = dependency.transform.outputs.apps_ad_group_principal_id
+  transform_developers_ad_group_principal_id = dependency.transform.outputs.developers_ad_group_principal_id
```

Comment:
```suggestion
  transform_feature_store_server_name                = dependency.transform.outputs.feature_store_server_name
  transform_feature_store_db_name                      = dependency.transform.outputs.feature_store_db_name
  transform_apps_ad_group_display_name           = dependency.transform.outputs.apps_ad_group_display_name
  transform_developers_ad_group_display_name = dependency.transform.outputs.developers_ad_group_display_name
  transform_apps_ad_group_principal_id               = dependency.transform.outputs.apps_ad_group_principal_id
  transform_developers_ad_group_principal_id     = dependency.transform.outputs.developers_ad_group_principal_id
```

Reply:
Check this alignment in VS Code as the GitHub editor is not ideal for this! Basically all the = signs should align in all HCL files not just TF ones

Reply:
Done 2d55b388b080110b76af6be5cbc5da3fda0343b8